openfisca_uk/parameters/benefits/LHA/means_test/income_disregard_lone.yaml

```diff
@@ -1,6 +1,6 @@
 description: Threshold in pounds per week from income for a lone parent, above which the Income Support amount is reduced equally.
```

Comment:
For consistency, could you rename this (and other `*_lone.yml` files) to `income_disregard_lone_parent.yml`? Looks to be how others are named.